app/controllers/agents/rdvs_controller.rb

```diff
@@ -25,7 +25,12 @@ def edit
 
   def update
     authorize(@rdv)
-    flash[:notice] = 'Le rendez-vous a été modifié.' if @rdv.update(rdv_params)
+    if params[:event] == 'cancel'
```

Comment:
This still doens't feel super restful: You cannot update a rdv by setting it's event to be 'cancel'.
Why not use params[:status] = 'excused' ? I know 'cancel' IS a better word for this, but since we have this as the status currently, I believe it's best to stick to it and not add new aliases that make reading the code ambiguous.